src/3d/p5.Renderer3D.Immediate.js

```diff
@@ -16,6 +16,9 @@
 var p5 = require('../core/core');
 var constants = require('../core/constants');
 
+var bezierDetail = 30;
```

Comment:
consider instead of hard coding the bezierDetail here, allowing the user to define bezierDetail with a short circuit fallback 
``` bezierDetail = bezierDetail || 30; ```
See primitives.js for more info on short circuit evaluation of vertex detail